config/base/ingress.yaml

```diff
@@ -1,17 +1,16 @@
-apiVersion: networking.k8s.io/v1
-kind: Ingress
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
 metadata:
-  name: cf-k8s-api-ingress
-  annotations:
-    kubernetes.io/ingress.class: gce
```

Comment:
Nice 😁 

I was playing around with our stuff today and was surprised to see this in play